spec/functional/resource/registry_spec.rb

```diff
@@ -112,19 +137,28 @@ def create_deletable_keys
     @node.name("windowsbox")
 
     @rest_client = double("Chef::ServerAPI (mock)")
-    allow(@rest_client).to receive(:create_url).and_return("reports/nodes/windowsbox/runs/#{@run_id}")
-    allow(@rest_client).to receive(:raw_http_request).and_return({ "result" => "ok" })
-    allow(@rest_client).to receive(:post_rest).and_return({ "uri" => "https://example.com/reports/nodes/windowsbox/runs/#{@run_id}" })
+
```

Comment:
These explicitly defined method stubs may be unnecessary, but leaving them in until the final PR at a minimum.